src/js/UnifileService.js

```diff
@@ -159,6 +159,7 @@ export default class UnifileService {
     return new Promise((resolve, reject) => {
       // Open a blank window right away, before we know the URL, otherwise the browser blocks it
       const win = window.open();
+      win.document.write(`Please wait while I check if you need to be redirected to a thind party service (${service}).`);
```

Comment:
I think you meant 'third' 😃

Reply:
thx :)
I should probably remove this or replace the message with a "loader"